assets/styles/common/_global.scss

```diff
@@ -32,39 +32,33 @@ h1, h2, h3, h4, h5, h6, p, label, span.label {
   font-family: $font-family;
   color: $color-neutral-80;
   font-weight: 400;
+  line-height: 1.2em;
 }
 
 h1 {
-  font-size: 32px;
-  line-height: 36px;
+  font-size: 48px;
 }
 
 h2 {
-  font-size: 28px;
-  line-height: 32px;
-  font-family: $font-family-slab;
+  font-size: 32px;
 }
 
 h3 {
-  font-size: 20px;
-  line-height: 24px;
+  font-size: 22px;
```

Comment:
> Sub-section titles should be H3 - 20px 
@bdolor Expected behavior says 20?

Reply:
I took the liberty of increasing `h3` to 22 and decreasing `h4` to 18 so that they had some visual distinction.